src/components/menu/styles.ts

```diff
@@ -18,3 +18,55 @@ export const ElMenuList = styled.div`
   background-color: var(--fill-white);
   box-shadow: 0px 4px 16px 0px #222b3329;
 `
+export const ElMenuItem = styled.li`
+  a,
+  button,
+  > div {
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: start;
+    height: var(--size-7);
+    padding: var(--spacing-none) var(--spacing-4);
+    gap: var(--spacing-4);
+    font-family: var(--font-family);
+    font-size: var(--font-size-sm);
+    font-weight: 400;
+    line-height: var(--line-height-sm);
+    letter-spacing: var(--letter-spacing-sm);
+    color: var(--text-primary);
+    &:hover {
+      color: var(--text-white);
+      background: var(--fill-action-dark);
+    }
+  }
+
+  /* customize html default button */
+  button {
+    background-color: transparent;
+    border: none;
+    border-radius: 0;
+  }
+`
+
+export const ElMenuItemGroupTitle = styled.span`
+  font-family: var(--font-family);
+  font-size: var(--font-size-2xs);
+  font-style: normal;
+  font-weight: 600;
+  line-height: var(--line-height-2xs);
+  letter-spacing: var(--letter-spacing-2xs);
+  color: var(--text-placeholder);
+  text-transform: uppercase;
+  display: flex;
+  height: 32px;
+  padding: var(--spacing-none, 0px) var(--spacing-4, 16px);
+  align-items: center;
+  align-self: stretch;
+`
+export const ElMenuItemGroup = styled.ul`
+  &,
+  & li > button {
+    width: 100%;
```

Comment:
**question:** Why is this the `& li > button` selector here and the `width: 100%` not part of the menu item styles? Also, why does it only apply to `li > button` descendants and not the other kinds of descendants our items are currently styling?

**suggestion**: This may be a personal preference, but I find:

```css
width: 100%;
```

much easier to understand than:

```css
& {
  width: 100%;
}
```

Reply:
> question: Why is this the & li > button selector here and the width: 100% not part of the menu item styles?

actually it could be, I just think since only the button need that style it would be less code than to put additional three line in the item `button { width: 100% }` 

> why does it only apply to li > button descendants and not the other kinds of descendants our items are currently styling?

its because only the button which is not full width, it appear to have fit content when I hover the button and the style applied

> suggestion: This may be a personal preference, but I find:

yes, same like  above I just thought that to minimize the code I just chain it with the list > button style, but as the current pr is small I think I can update it, sorry for this😄

Reply:
Yeah, okay. It's really important we don't decide what unit of code is responsible for what piece of logic or styling based on lines of code. We want components to have the correct responsibilities, and we want code to be readable by others (which may mean more lines of code, not less).

Reply:
I will update it👍